addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -643,7 +643,46 @@ registry.share = publicWidget.Widget.extend({
     },
 });
 
-registry.mediaVideo = publicWidget.Widget.extend({
+const MobileYoutubeAutoplayMixin = {
+    _setupAutoplay: function (src) {
```

Comment:
Missing documentation for both methods. No need of huge texts but at least adding `@private` and `@param` for convention.